src/Slides.js

```diff
@@ -0,0 +1,29 @@
+import { URL } from 'url';
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  if (!['slides.com', 'www.slides.com', 'team.slides.com'].includes(host)) {
+    return false;
+  }
+
+  const pathTrimmed = pathname.split('/').filter(Boolean);
+  if (pathTrimmed.length !== 2) {
```

Comment:
This can be simplified to

```ts
return pathTrimmed.length === 2;
```

Because
```ts
if(!condition) {
  return true;
}

return false;
```
is the same as
```ts
return !condition
```